Gemfile

```diff
@@ -15,6 +15,7 @@ gem 'jbuilder', '~> 2.7'
 gem 'jwt'
 gem 'kaminari'
 gem 'monetize'
+gem 'ocso_scraper', git: 'https://github.com/AyOK-Code/ocso_scraper', ref: '4023411'
```

Comment:
this wasn't supposed to come off

Reply:
got it just adding osco back to the gemfile